backend/views/core/invoices/view.py

```diff
@@ -48,10 +54,10 @@ def view(request, uuid):
         messages.error(request, "Invoice not found")
         return redirect("index")
 
-    try:
+    if request.user != AnonymousUser():
         currency_symbol = request.user.user_profile.get_currency_symbol()
```

Comment:
We could actually use invoice.user because otherwise the value might be in the wrong currency. For example if the users currency is ¥ but the customers is defaulted to £ it'll be the wrong "amount" or needs to be converted

Reply:
fixed